app/scripts/controllers/add-events.js

```diff
@@ -35,7 +35,7 @@ module.exports = function (Api, $scope, $routeParams, $location) {
   Api.Patients.get({ id: $routeParams.patientId }, success, error)
 
   $scope.events = [
-    { code: 'sample-event', display: 'Sample Event', formName: 'FormBuilderaddEventSampleEvent' }
+    { code: 'birth', display: 'Birth', formName: 'FormBuilderAddEventBirth' }
```

Comment:
Lets update the display name to "Birth Registration" to give more of a description what this event is form, Same applies to the form heading